app/main/views/agreement.py

```diff
@@ -34,7 +32,6 @@ def service_agreement(service_id):
 @user_has_permissions('manage_service')
 def service_download_agreement(service_id):
     return send_file(**get_mou(
-        current_service.organisation.crown_status_or_404
     ))
```

Comment:
[suggestion (nonblocking)] could we move these closing parens onto the single line?